### DISCUSSION TITLE
Request for Guidance on Labeling Different Brain Regions Over Time in Neuronal Data Analysis

### DISCUSSION THREAD

**Discussion by qweasdf1354** [AdaptiveMotorControlLab/CEBRA#146] [Q&A] [answered]
Dear Cebra Community,

I hope you are all doing well. I am reaching out to seek your expert advice on a challenging aspect of my research. I am currently analyzing neuronal data from multiple brain regions and am trying to figure out the best approach to label these regions across different time points.

My approach so far involves dimensionality reduction on neuronal recordings from each region. To illustrate, suppose I have data from two regions, resulting in matrices A and B with dimensions 50x1000 and 48x1000 (Neurons x time), respectively. My plan is to perform PCA on each matrix individually, extracting the first 48 principal components (to match the neuron count of the smaller region).

Following dimensionality reduction, I propose to horizontally stack the PCA-transformed matrices of A and B, resulting in a combined matrix of 98x2000. This matrix would then be labeled with categorical variables that reflect the time points associated with each brain region (e.g., '1' for the first 1000 columns from region A, and '2' for the last 1000 columns from region B).

This method is inspired by techniques used in single-cell data analysis, and I am curious if it is suitable for my current project. I would greatly appreciate any insights or alternative methods you might recommend for preprocessing data from two different brain regions, to ensure consistency in cell dimensions and counts.

Thank you very much for considering my inquiry. I look forward to your valuable suggestions.

Warm regards,
Zhou

**Comment by qweasdf1354** [comment]
Sorry for mis 98x2000 should be 48+48=96 and it should be 96*2000

**Reply by MMathisLab** [reply, marked as answer]
Hi Zhou, I feel this question goes way beyond our code base, and seems an ideal discussion with your PI/lab. We can't consult on experimental design (it's a lot of work, and many more details need considered).

**Reply by qweasdf1354** [reply]
Thanks for quick response, I will read more article related and take them into consideration.
